<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabZigbee">
     <attribute name="title">
      <string>Zigbee</string>
     </attribute>
     <widget class="QGroupBox" name="gb_SettingZb">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Setting</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>171</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="SettingLayoutZb">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>13</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_DeviceZb">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_DeviceZb"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lb_BaudRateZb">
          <property name="text">
           <string>Baudrate:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lb_DataSizeZb">
          <property name="text">
           <string>Data size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbt_OpenZb">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cb_BaudrateZb"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cb_DatasizeZb"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="ck_ParityZb">
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Display data</string>
      </property>
      <widget class="QPlainTextEdit" name="pte_DisplayDataZb">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>231</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>61</width>
         <height>33</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="1">
         <widget class="QPushButton" name="btClear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lb_CmdZb">
         <property name="text">
          <string>Cmd:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_CmdZb"/>
       </item>
       <item>
        <widget class="QLabel" name="lb_ModeZb">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_ModeZb"/>
       </item>
       <item>
        <widget class="QLabel" name="lb_NodeZb">
         <property name="text">
          <string>Node:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_NodeZb"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>40</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="pbt_SendCmdZb">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabView">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>271</width>
        <height>35</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Total spots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTotalSpots"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Blank spots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leBlankSpots"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>131</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>End spots</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Begin spots</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSpinBox" name="sbBeginSpots"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QSpinBox" name="sbEndSpots"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QRadioButton" name="rbViewAll">
         <property name="text">
          <string>View all</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="btView">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Table view spots</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>271</width>
         <height>201</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>201</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
        <property name="background">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
